dart/test/sentry_event_test.dart

```diff
@@ -134,18 +130,7 @@ void main() {
               },
             ]
           },
-        }..addAll(
```

Comment:
don't we test this anymore?

Reply:
as the tested `throwable` is an error, it should contain its stacktrace, so I'm not sure it make sense to set the event.stacktrace in this case ?

Reply:
it doesnt, but lets make sure that this is use case is tested elsewhere (throwable not being an error), or create a new test for it

Reply:
✅